lib/libprocstat/libprocstat.c

```diff
@@ -355,6 +355,52 @@ procstat_freeprocs(struct procstat *procstat __unused, struct kinfo_proc *p)
 	p = NULL;
 }
 
+int
+procstat_getc18n(struct procstat *procstat, struct kinfo_proc *kp,
+    struct rtld_c18n_stats *stats)
+{
+	int name[4];
```

Comment:
Can't these have constants?  Both here and below where you reference each part of the array?

Reply:
Using `int name[4]` as the argument for `sysctl(3)` seems to be a common pattern. There are many similar cases in the same file.